edkahara/iReporter
# [finishes #162216820] create a get all red flags api endpoint

__What does this PR do?__
Create a get all red-flags api endpoint.

__Description of tasks to be completed.__
Create an api endpoint to get all red-flags.

__How should it be manually tested?__
Clone the project. Check out the ft-create-get-all-red-flags-api-endpoint-#162216820 branch. Start the environment using env\Scripts\activate. Run flask run on the terminal and go to Postman. To perform the tests run nose2.

__Relevant pivotal tracker stories.__
#162216820
